extensions/stackdriver/log/logger.cc

```diff
@@ -176,6 +177,8 @@ bool fillAuthInfo(const std::string& response_details,
         absl::StrCat(policy_namespace, ".", policy_name);
     (*label_map)["policy_rule"] = policy_rule_index;
     return true;
+  } else if (RE2::PartialMatch(response_details,rbac_denied_match_default)){
```

Comment:
Should return true inside?

Reply:
Also no need to `else if`? Just `if (...)`.

Reply:
Thanks..Fixed